crates/miden-lib/asm/kernels/transaction/lib/epilogue.masm

```diff
@@ -294,6 +314,10 @@ export.finalize_transaction.4
     exec.execute_auth_procedure
     # => []
 
+    # update the account code commitment in case its code has been changed
+    exec.update_code_commitment
```

Comment:
Not for this PR, but I'm wondering how does this work with account delta computation. We probably want account delta to include commitment to the new code, but I'm not sure it currently does. Let's create an issue for this.

Reply:
I think that would be handled as part of updating account code generally, but I can't find an issue for that either, so created https://github.com/0xMiden/miden-base/issues/1695.